src/compiler/crystal/compiler.cr

```diff
@@ -18,6 +18,7 @@ module Crystal
   # optionally generates an executable.
   class Compiler
     CC = ENV["CC"]? || "cc"
+    CL = "cl"
```

Comment:
Yeah that's a good idea. I want to work out an env var or commandline flag to set the linker, like CC on posix.